src/routes/index.js

```diff
@@ -1,5 +1,13 @@
-const router = require("express").Router();
+import express from 'express';
 
+const router = express.Router();
+
+router.get("/", (req, res) => {
+    res.status(200).json({
+        status: "success",
+        body: "Welcome to Barefoot Nomad API"
+    });
+});
 router.use("/api", require("./api"));
```

Comment:
```suggestion
router.use("/api", "api"));
```
As earlier this must have been imported before this can be used. remove the `require` keyword for now **OR** Just comment it out